resources/prosody-plugins/mod_muc_census.lua

```diff
@@ -0,0 +1,73 @@
+-- report list of rooms with # of members and created date, filtered on test rooms
```

Comment:
Can you add a description of where this module needs to be loaded, is it under the main virtual host or the muc component?